openml-lightgbm/lightgbm-provider/src/test/java/com/feedzai/openml/provider/lightgbm/TestSchemas.java

```diff
@@ -202,4 +202,28 @@
             0,
             TestSchemas.NUMERICALS_SCHEMA_WITH_LABEL_AT_END.getPredictiveFields()
     );
+
+    /**
+     * Schema with a feature with non-ASCII characters.
+     */
+    static final DatasetSchema SCHEMA_WITH_ONE_NON_ASCII_FEATURE = new DatasetSchema(
+            0,
+            ImmutableList.<FieldSchema>builder()
+                    .addAll(NUMERICALS_SCHEMA_WITH_TARGET_AT_START.getFieldSchemas())
+                    .add(new FieldSchema("cão", 5,
+                            new NumericValueSchema(false)))
+                    .build()
+    );
+
+    /**
+     * Schema with two feature with non-ASCII characters.
```

Comment:
typo: `features`

I think you mean one feature with two non ASCII characters. If so, change it to
`Schema with a feature that contains two non ASCII characters`
Similar to the above one